tests/ui/resolve/issue-82865.stderr

```diff
@@ -15,6 +15,16 @@ LL |     Box::z
 LL |     mac!();
    |     ------ in this macro invocation
    |
+note: if you're trying to build a new `Box<_, _>` consider using one of the following associated functions:
```

Comment:
I think we need to limit the suggestions here -- if there are more than a couple of functions that yield the type, listing all of them is not helpful.

Reply:
we filter on 8, I can reduce that to 4?

Reply:
Reduced to 4.